File: GENOR/Client/Assets/Scripts/Models/Indicator.cs
using System.Collections;
using System.Collections.Generic;

public class Indicator
{
    public string name { get; private set; }
    public float weight { get; private set; }

    public Indicator()
    {}
    
    public Indicator(string name, float weight): this()
    {
        this.name = name;
        this.weight = weight;
    }
}

File: Mobile and desktop applications/Mobile application/Walkability/app/src/main/java/com/example/walkability/models/Indicator.kt
package com.example.walkability.models

import java.io.Serializable

class Indicator (
    val name: String,
    val description: String = "",
    val group: String = "",
    val datasetName: String = "",
    var weight: Float = 100.0f,
    val parameterLabel: String = "",
    val defaultParameterValue: String = "",
    var parameterValue: Float = -1f): Serializable